src/sidebar/store/modules/toast-messages.js

```diff
@@ -7,6 +7,7 @@ import { createStoreModule, makeAction } from '../create-store';
  * @prop {string} message
  * @prop {string} moreInfoURL
  * @prop {boolean} isDismissed
+ * @prop {boolean} [visuallyHidden=false]
```

Comment:
It might be better to make this a required property of `ToastMessage`, so that any consumers of the `getMessage` selector can rely on it being set. The `ToastMessages` service handles populating this field with a default value, as it does for `isDismissed`.